### ISSUE TITLE
context is not defined - showPaths issue

### ISSUE THREAD

**Issue by mfld-pub** [nttgin/BGPalerter#302]
**Describe the bug**
I am testing the v 1.26.0 release and configured Telegram notifications and would like some clarification on the 
`chatIds:` and `default: ` options in the configuration.

**Provide an example**
 ```
 - file: reportTelegram
channels: 
- hijack
- newprefix
- visibility
- path
- misconfiguration
- rpki
params:
showPaths: 5 # Amount of AS_PATHs to report in the alert
botUrl: https://api.telegram.org/bot12345678:XXXXXXXXXXXX/sendMessage         <- Bot ID and API token. 
Validated via _getUpdates_ endpoint                                                                                                                                      
chatIds: 
default: xxxxx <- ID of the group chat that the bot and the users are in.
Bot can see messages posted there via the /getUpdates endpoint        

```
**Expected behavior**
I would expect that events are being sent to the chat ID of the default user group. In prefixes.yml I do have the monitored ASN allocated to the "default" group. 
```
options:
  monitorASns:
    '212996':
      group: default
```
E-mail alerts work OK.

I have **chatIds** blank because I just want it to send to the default group 

**Are you using the binary or the source code?**
The binary on Ubuntu 20.04 as a systemd service.

**Your information**
Michael Felden, AS212996

**Comment by massimocandela** [comment]
Hello @mfld-pub,

The error you reported in the subject makes me think the indentation of your config file is corrupted.
Could you verify that your indentation is as follows:

```
  - file: reportTelegram
    channels:
      - hijack
      - newprefix
      - visibility
      - path
      - misconfiguration
      - rpki
    params:
      showPaths: 0 # Amount of AS_PATHs to report in the alert
      botUrl: https://api.telegram.org/bot<_BOT_ID_>/sendMessage
      chatIds:
        default: _CHAT_ID_
```
Indeed, the default groups is mandatory and will receive all types of alerts. The default groups must be declared inside chatIds (it is the id of the default group) and cannot be blank.

**Comment by mfld-pub** [comment]
Hi Massimo,

indentation was some sort of clipboard issue into GitHub, actual file has it as:
![bug2](https://user-images.githubusercontent.com/66289900/88286050-bd7eb000-cd22-11ea-86b7-4e5b2bb9f40f.JPG)

So the `default:` was messed up. I will fix it now and report back! Thanks for spotting this.

**Comment by mfld-pub** [comment]
I withdrew a route to trigger an alert. 
Pushover and e-mail alert worked. Telegram did not deliver. error log file shows no issue this time. I was running in foreground this time to see what is going on and it threw these errors:

```
ReferenceError: context is not defined
    at reportTelegram._sendHTTPMessage (/snapshot/BGPalerter/build/src/reports/reportHTTP.js:66:72)
    at reportTelegram.report (/snapshot/BGPalerter/build/src/reports/reportHTTP.js:101:21)
    at /snapshot/BGPalerter/build/src/reports/report.js:184:22
    at /snapshot/BGPalerter/build/src/utils/pubSub.js:45:13
    at new Promise (<anonymous>)
    at _loop (/snapshot/BGPalerter/build/src/utils/pubSub.js:44:11)
    at PubSub.publish (/snapshot/BGPalerter/build/src/utils/pubSub.js:51:11)
    at MonitorVisibility._publishOnChannel (/snapshot/BGPalerter/build/src/monitors/monitor.js:206:18)
    at MonitorVisibility._publishGroupId (/snapshot/BGPalerter/build/src/monitors/monitor.js:188:13)
    at MonitorVisibility.publishAlert (/snapshot/BGPalerter/build/src/monitors/monitor.js:105:13)
```

**Comment by massimocandela** [comment]
I found where the issue is for the latest message.
I'm patching it now, thanks for reporting. I will probably issue a minor release.

In the meanwhile, disable showPaths by setting it to 0. Thanks again for reporting.

**Comment by mfld-pub** [comment]
Thank you! I can confirm it is working now with `showPaths: 0 `

**Comment by massimocandela** [comment]
Released patch in [v1.26.1](https://github.com/nttgin/BGPalerter/releases/tag/v1.26.1).
I created tests to cover the issue. We don't use the showPaths feature so I didn't notice it before. Sorry for the inconvenience.
